classes/local/upload_helper.php

```diff
@@ -76,10 +76,14 @@ public static function get_upload_jobs($courseid) {
 
         $allnamefields = get_all_user_name_fields(true, 'u');
 
-        $sql = "SELECT uj.*, $allnamefields, md.metadata
+        $sql = "SELECT uj.*, $allnamefields, md.metadata,
+                f1.filename as presenter_filename, f1.filesize as presenter_filesize,
+                f2.filename as presentation_filename, f2.filesize as presentation_filesize
                 FROM {block_opencast_uploadjob} uj
                 JOIN {user} u ON uj.userid = u.id
-                JOIN {block_opencast_metadata} md ON uj.id = md.uploadjobid 
+                JOIN {block_opencast_metadata} md ON uj.id = md.uploadjobid
```

Comment:
https://github.com/unirz-tu-ilmenau/moodle-block_opencast/pull/91#discussion_r315761086

Reply:
What did you want to tell me with the linked comment? I wanted to rewrote the code here to get rid of the filename and filesize being part of the uploadjob table.